src/Storage/Storage.Management/StorageAccount/GetAzureStorageBlobInventoryPolicy.cs

```diff
@@ -95,9 +94,8 @@ public override void ExecuteCmdlet()
                     break;
             }
 
-            BlobInventoryPolicy policy = this.StorageClient.BlobInventoryPolicies.Get(
-                 this.ResourceGroupName,
-                 this.StorageAccountName);
+            Track2.BlobInventoryPolicyResource policy =
+                this.StorageClientTrack2.GetBlobInventoryPolicyResource(this.ResourceGroupName, this.StorageAccountName, "default").Get();
```

Comment:
>"default" [](http://example.com/codeflow?start=121&length=9)

It is recommended to define a static string and name it something like "defaultBlobInventoryPolicyName".

It would be easier for everyone to understand and review this parameter better.

Reply:
Thanks! Done

Reply:
Sorry totally missed this comment. Will include this change in the next PR. Thanks a lot!